src/darker/tests/test_main.py

```diff
@@ -22,14 +22,19 @@ def test_isort_option_without_isort(tmpdir, without_isort, caplog):
 
 
 @pytest.fixture
-def run_isort(git_repo, monkeypatch, caplog):
+def run_isort(git_repo, monkeypatch, caplog, request):
+    from black import find_project_root
```

Comment:
Here, too, maybe move import to top if there's no reason to have a local import?